gatsby-config.js

```diff
@@ -23,9 +25,18 @@ module.exports = {
         includeCollections: ["shop", "content"],
       },
     },
+    {
+      resolve: "gatsby-plugin-google-analytics",
+      options: {
+        // replace "UA-XXXXXXXXX-X" with your own Tracking ID
+        trackingId: "UA-XXXXXXXXX-X",
+      },
+    },
   ],
   siteMetadata: {
     title: "Netlify Swag Store",
```

Comment:
```suggestion
    title: "Netlify Swag Store",
    metatitle: "Netlify Store - Awesome Apparel, Stickers, and Other Swag",
```

ahh now seeing below in file layout here that `title` this ties into `seo.js`
adding new `metatitle` field then for homepage

Reply:
realize this would cause some refactoring - if not able to do so, we can scratch the suggestion altogether, no worries.